src/server/sched_func.c

```diff
@@ -439,9 +439,11 @@ set_sched_default(pbs_sched* psched)
 	}
 	if ((psched->sch_attr[(int) SCHED_ATR_sched_state].at_flags & ATR_VFLAG_SET) == 0) {
 		psched->sch_attr[(int) SCHED_ATR_sched_state].at_val.at_str = malloc(SC_STATUS_LEN + 1);
-		if (psched->sch_attr[(int) SCHED_ATR_sched_state].at_val.at_str == NULL)
+		if (psched->sch_attr[(int) SCHED_ATR_sched_state].at_val.at_str == NULL) {
+			snprintf(log_buffer, LOG_BUF_SIZE-1, "malloc failure (errno %d)", errno);
+			log_err(PBSE_SYSTEM, __func__, log_buffer);
```

Comment:
The first argument to log_err() is the errno, not a PBSE error number.  If you want to log at PBSE_SYSTEM, you'll need to use a log_event().  You don't need to embed the errno as part of the message, use it as the first argument here.  Same goes for the rest of the messages like this one.

Reply:
Done